build_image.sh

```diff
@@ -15,14 +15,14 @@ fi
 setup_type=$1
 app_path=$PWD
 app_name=${app_path##*/}
-parent_path=`dirname $PWD`
+#parent_path=`dirname $PWD`
 
 echo "-- Generating image for $setup_type" container.
 
 mkdir -p $app_path/workspace/docker_certificates
 
 echo "-- Copying host config file to be used by docker in current directory"
-cp $parent_path/config/cchost_config_${setup_type}_js.json $app_path/workspace/docker_certificates/.
+cp $app_path/config/cchost_config_${setup_type}_js.json $app_path/workspace/docker_certificates/.
```

Comment:
One of the reasons for us to move the files is to avoid this step.

Once we have the configuration files inside each application, we will be able to refer to them directly, without the need of copying them to this common folder.

Docker files have to change as well for this to work. (More explanation comment on the docker files.)